my-app/src/public-components/footer/footer.js

```diff
@@ -0,0 +1,112 @@
+
+import React, { Component } from 'react';
+import { Icon } from 'antd';
+import classnames from "classnames";
+
+import { Textlink } from '../text-link'
+import classes from './footer.css';
+
+
+export class Footer extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      prevScrollpos: window.pageYOffset,
+      visible: true
+    };
+
+    this.handleScroll = this.handleScroll.bind(this);
```

Comment:
if you use handleScroll like arrow function you don't need bind it. Remove this line.